### PR TITLE
Syminner

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: mat64/inner.go [hunk 1]
```diff
@@ -27,6 +27,17 @@ func Inner(x []float64, A Matrix, y []float64) float64 {
 	var sum float64
 
 	switch b := A.(type) {
+	case RawSymmmetricer:
```
Comment (fhs) [inline mat64/inner.go]: There seems be an extra 'm' in the interface name.

Also, we have an interface named Symmetric, which seems inconsistent with RawSymmetricer. Interface names are usually `<verb>-er` or `<noun>`, so I would favor RawSymmetric.

Comment (btracey) [inline mat64/inner.go]: Thanks for the catch

This particular case is weird. The main matrix-type interfaces are Matrix and Symmetric. All of the other interfaces follow the normal _er convention, including RawMatrixer. RawSymmetricer follows that precedent.

**Comment by btracey** [comment]
PTAL

**Comment by kortschak** [comment]
LGTM

**Pull request merged**